README.md

```diff
@@ -102,13 +102,13 @@ You'll be able to load the samples by visiting `https://localhost:8000/packages/
 
 ### Available Samples
 
-| Sample | App Link | Source |
-| ------ | -------- | ------ |
-| Implicit Grant Flow | [local app](https://localhost:8000/packages/node_modules/samples/browser-auth-implicit) | [code](./packages/node_modules/samples/browser-auth-implicit) |
-| Single Party Calling | [local app](https://localhost:8000/packages/node_modules/samples/browser-single-party-call) | [code](./packages/node_modules/samples/browser-single-party-call) |
+| Sample                         | App Link                                                                                              | Source                                                                      |
```

Comment:
Any reason for these changes?

Reply:
I think my linter went ahead and autofixed when i saved